src/DataCollector/DoctrineDataCollector.php

```diff
@@ -305,13 +307,17 @@ public function getGroupedQueries(): array
                 $totalExecutionMS += $query['executionMS'];
             }
 
+            $connectionGroupedQueries = array_values($connectionGroupedQueries);
+
             usort($connectionGroupedQueries, static function ($a, $b) {
                 if ($a['executionMS'] === $b['executionMS']) {
                     return 0;
                 }
 
                 return $a['executionMS'] < $b['executionMS'] ? 1 : -1;
             });
+            /** @var list<array{executionMS: float, explainable: bool, sql: string, params: array<mixed>|null, runnable: bool, types: ?array<array-key, Type|int|string|null>, count: int, index: int, executionPercent?: float}> $connectionGroupedQueries */
+            $connectionGroupedQueries          = $connectionGroupedQueries;
```

Comment:
I removed this and rerun phpstan. It has no effect on result from Phpstan. Why did you add it?

Reply:
Removed and fixed at e4b1402f